Update/add search props (#4990)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/index.mdx

description: "Learn more about Search, which is available in several Sentry features."
---

> Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues** and **Discover**.

Reviewer: ```suggestion
Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
```

The token `example error` is utilizing the optional raw search and is passed as part of the issue search query (which uses a CONTAINS match similar to SQL). When using the optional raw search, you can provide _one_ string, and the query uses that entire string.

> Search terms should auto-complete, and when they don't, that means you have an invalid data set. If you complete the term and tap the Return/Enter key, an error message is displayed.

Reviewer: Thoughts on something like this? most of the UI doesn't allow picking dataset
```suggestion
Search terms should auto-complete, and when they don't, that means your term is incompatible with the dataset. If you complete the term and tap the Return/Enter key, an error message is displayed.
```
